install.md

```diff
@@ -8,6 +8,8 @@
 sudo pacman -S podman
 ```
 
+If you have problems when running podman in [rootless](rootless.md) mode follow [these instructions](https://wiki.archlinux.org/index.php/Linux_Containers#Enable_support_to_run_unprivileged_containers_(optional))
```

Comment:
I'd not include the link to rootless.md.

Reply:
Just to make sure I understand your comment: you want me to remove that link, or is this just an expression of opinion that I'm free to (not) consider?

Reply:
I would like it to be removed unless you or others have strong differing opinions.  That page doesn't tell you how to setup rootless or what it's about, but rather shows you the work that is still outstanding for it.

Reply:
If you can wait, I'd be hapy if you changed the rootless link to the rootles section being added to the README.md in #3038